protos/fpc/key_dist.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package key_distribution;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CCKeyRegistrationMessage {
+    bytes cc_params_hash = 1;       // defines the context as SHA2 of cc_params
+    bytes chaincode_ek   = 2;       // public chaincode encryption key
+    bytes enclave_id     = 3;       // creator of this message
+}
+
+message SignedCCKeyRegistrationMessage {
+    bytes msg_bytes = 1;            // public chaincode encryption key
```

Comment:
shouldn't this also be ideally a `Any` (of CCKeyRegistrationMessage)? And comment seems wrong ...
```suggestion
    google.protobuf.An serialized_cckey_reg_msg = 1;            // serialization of type CCKeyRegistrationMessage 
```